dayrepo/snippets/views.py

```diff
@@ -197,26 +199,28 @@ def post(self, request):
     
 checklist_post = ChecklistView.as_view()
 
-import openpyxl
-
-    
-def excelfile_download(request):
+# Excelファイル出力    
+def excelfile_download(request,snippet_id):
     """
     Excel output from template
     """
+    pk=snippet_id
+    print("ぴーけー",pk)
+    # listのボタンで選択したSnippetデータを変数で受け取る
```

Comment:
クライアントの実装を気にしているコメントはあまり芳しくないんだが、これでいかが？
```suggestion
    # 選択したSnippetデータを変数で受け取る
```

Reply:
出来ればもう少し選択に対するヒントが欲しいです。
どこで何をどうしたものかという情報を書いたつもりなので。

Reply:
では# listページで選択したSnippetデータを変数で受け取る
でいいかな？

Reply:
> listページで選択した

の部分が、バックエンド側としては関係がないって話しだねぇ

Reply:
ではこれでいいかな？
# 引数で受け取った値を変数に代入

Reply:
Snippet である旨もなくなって、よりコメントの必要性を感じないコードみりゃわかるやん状態にはなってしまったが 笑、とりあえずクライアント依存はなくなったのであとはお好きにどうぞ！笑

Reply:
コードが確認できず、修正されたのかわからないのでとりま残します〜

Reply:
[commit](https://github.com/bob-g12/dayrepo-for-transportation/pull/51/commits/fe24403a67263596c67e7de01b8843375a544a74)